virtcontainers/network.go

```diff
@@ -579,6 +580,51 @@ func newNetwork(networkType NetworkModel) network {
 	}
 }
 
+func getNetNsFromBindMount(nsPath string) (string, error) {
```

Comment:
You could add a unit test for this function if you made `mountInfoFile` a global `var` (that the tests could modify) or an extra param to this function.